package.json

```diff
@@ -36,9 +36,10 @@
   "browser": "./dist/otpauth.esm.js",
   "exports": {
     ".": {
-      "types": "./dist/otpauth.d.ts",
-      "bun": "./dist/otpauth.esm.js",
-      "deno": "./dist/otpauth.esm.js",
```

Comment:
Could you please add the `bun` and `deno` entries again? I will evaluate their need later, but I would like to keep this PR without unrelated changes.